packages/styletron-standard/package.json

```diff
@@ -30,7 +31,7 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
-    "@rtsao/csstype": "2.6.5-forked.0",
+    "csstype": "2.6.9",
```

Comment:
This may create some cryptic flow error messages and was why we added the flow error message tests

Reply:
Sorry, we had attempted to update it a few months ago but the newer version presented styletron users with hard-to-comprehend error messages. Just pointing out that we may find issues in this area